hocon/src/main/scala/com/avsystem/commons/hocon/ConfigCompanion.scala

```diff
@@ -0,0 +1,69 @@
+package com.avsystem.commons
+package hocon
+
+import com.avsystem.commons.meta.MacroInstances
+import com.avsystem.commons.misc.ValueOf
+import com.avsystem.commons.serialization.{GenCodec, GenKeyCodec, GenObjectCodec, HasGenObjectCodecWithDeps}
+import com.avsystem.commons.serialization.GenCodec.ReadFailure
+import com.typesafe.config.{Config, ConfigFactory, ConfigObject}
+
+import scala.concurrent.duration.*
+
+trait CommonsHoconCodecs {
```

Comment:
nit, naming: we already have `CborOptimizedCodecs` and `BsonGenCodecs`. Maybe we should try to stay consistent with one of those in naming (probably the latter).

Reply:
Renamed to `HoconGenCodecs`